sources/web/datalab/polymer/components/file-browser/file-browser.ts

```diff
@@ -401,44 +409,60 @@ class FileBrowserElement extends Polymer.Element implements DatalabPageElement {
   /**
    * Calls the FileManager to get the list of files at the current path, and
    * updates the _fileList property.
+   * This method can be called multiple times, and it will ignore the fetch
+   * result if the currentFile object has changed after the request was made.
```

Comment:
This comment is not quite accurate. It will ignore the fetch result if the currentFile object does not match the file for the request. It could have changed after the request was made, and then changed back, and the request won't be ignored. But perhaps that's too much detail to worry about here.

Reply:
Yea, I too didn't think it was worth explaining in more details, the word `changed` captures the common meaning.
I made it a little clearer.